docs/wcs/examples/from_file.py

```diff
@@ -40,6 +41,22 @@ def load_wcs_from_file(filename):
     # some floating-point error.
     assert np.max(np.abs(pixcrd - pixcrd2)) < 1e-6
 
+    # As illustrated below, origin defines whether the origin pixel position
+    # is 0- or 1-indexed.
+    x_origin = 0
+    y_origin = 0
```

Comment:
I wonder if because of this someone would think that `x` and `y` can have different origin, i.e. one is 0-based the other 1-based. I think that the naming is confusing, i.e. having `origin` in the name of the inputs. I'd prefer these to be called simply `x` and `y`.

`x, y = 0, 0`

Reply:
Done.